server/src/test/java/org/apache/druid/server/coordinator/TestDruidCoordinatorConfig.java

```diff
@@ -252,4 +223,333 @@ public boolean getCoordinatorKillIgnoreDurationToRetain()
+    }
+
+    public Builder withCoordinatorRuleKillPeriod(Duration coordinatorRuleKillPeriod)
+    {
+      this.coordinatorRuleKillPeriod = coordinatorRuleKillPeriod;
+      return this;
+    }
+
+    public Builder withCoordinatorRuleKillDurationToRetain(Duration coordinatorRuleKillDurationToRetain)
+    {
+      this.coordinatorRuleKillDurationToRetain = coordinatorRuleKillDurationToRetain;
+      return this;
+    }
+
+    public Builder withCoordinatorDatasourceKillPeriod(Duration coordinatorDatasourceKillPeriod)
+    {
+      this.coordinatorDatasourceKillPeriod = coordinatorDatasourceKillPeriod;
+      return this;
+    }
+
+    public Builder withCoordinatorDatasourceKillDurationToRetain(Duration coordinatorDatasourceKillDurationToRetain)
+    {
+      this.coordinatorDatasourceKillDurationToRetain = coordinatorDatasourceKillDurationToRetain;
+      return this;
+    }
+
+    public Builder withLoadTimeoutDelay(Duration loadTimeoutDelay)
+    {
+      this.loadTimeoutDelay = loadTimeoutDelay;
+      return this;
+    }
+
+    public Builder withLoadQueuePeonRepeatDelay(Duration loadQueuePeonRepeatDelay)
+    {
+      this.loadQueuePeonRepeatDelay = loadQueuePeonRepeatDelay;
+      return this;
+    }
+
+    public Builder withLoadQueuePeonType(String loadQueuePeonType)
+    {
+      this.loadQueuePeonType = loadQueuePeonType;
+      return this;
+    }
+
+    public Builder withHttpLoadQueuePeonRepeatDelay(Duration httpLoadQueuePeonRepeatDelay)
+    {
+      this.httpLoadQueuePeonRepeatDelay = httpLoadQueuePeonRepeatDelay;
+      return this;
+    }
+
+    public Builder withCuratorLoadQueuePeonNumCallbackThreads(int curatorLoadQueuePeonNumCallbackThreads)
+    {
+      this.curatorLoadQueuePeonNumCallbackThreads = curatorLoadQueuePeonNumCallbackThreads;
+      return this;
+    }
+
+    public Builder withHttpLoadQueuePeonHostTimeout(Duration withHttploadQueuePeonHostTimeout)
+    {
+      this.httpLoadQueuePeonHostTimeout = httpLoadQueuePeonHostTimeout;
+      return this;
+    }
+
+    public Builder withHttpLoadQueuePeonBatchSize(int httpLoadQueuePeonBatchSize)
+    {
+      this.httpLoadQueuePeonBatchSize = httpLoadQueuePeonBatchSize;
+      return this;
+    }
+
+    public Builder withCompactionSkippedLockedIntervals(boolean compactionSkippedLockedIntervals)
+    {
+      this.compactionSkippedLockedIntervals = compactionSkippedLockedIntervals;
+      return this;
+    }
+
+    public Builder withCoordianatorAuditKillPeriod(Duration coordinatorAuditKillPeriod)
+    {
+      this.coordinatorAuditKillPeriod = coordinatorAuditKillPeriod;
+      return this;
+    }
+
+    public Builder withCoordinatorAuditKillDurationToRetain(Duration coordinatorAuditKillDurationToRetain)
+    {
+      this.coordinatorAuditKillDurationToRetain = coordinatorAuditKillDurationToRetain;
+      return this;
+    }
+
+    //TODO add mising fields to both TEst and builder.build investigate loadTimeoutDelay naming in Test. Migrate usage to builder.
```

Comment:
Unresolved TODO?

Reply:
ah, I actually did the work but forgot to remove my reminder. deleted.